### ISSUE TITLE
Visualizated explanations of the examples

### ISSUE THREAD

**Issue by johnny5550822** [Element-Research/rnn#250]
I found the examples founder is great. Have you have any plan to further develop visual explanation on every example file? For example the "recurrent-time-series.lua", when we go over the code, explanation + figures may help the user to understand not only how rnn package works, but also the basic of the recurrent network structure?

**Comment by nicholas-leonard** [comment]
@johnny5550822 Yeah that kind of tutorial would be awesome. Its just that I have been a little overwhelmed with other stuff. Tutorials take time to do. Would you be open to writing one?

Otherwise, I am working on a blog post / tutorial for the noise-contrastive-estimate.lua example.

**Comment by rracinskij** [comment]
Trying to understand is that kind of stuff is compatible with rnn's - http://homes.cs.washington.edu/~marcotcr/blog/lime/
If yes, it could eventually add some valuable insights to the rnn mechanics at some later point.

**Comment by johnny5550822** [comment]
@nicholas-leonard Yes, I am thinking about it. If I make one for some examples, I will send you the link. A tutorial will boost the good codes of RNN.

@rracinskij Thanks. Yes, I agree with it. Sometime, there are some tutorails out there, but it is hard to google them~

**Comment by nicholas-leonard** [comment]
@johnny5550822 Awesome! Keep us posted :)
